README.md

```diff
@@ -1,6 +1,6 @@
-# Scaffold for social media app with Ruby on Rails
+# Stay in Touch
 
-> This repo includes intial code for social media app with basic styling. Its purpose is to be a starting point for Microverse students.
+This is a social media app in which users can create posts, read posts of different users, comment, like, dislike, send and accept friendship invitaitons. 
 
```

Comment:
[OPTIONAL]
- Would be great to have an image of you ERD diagram into your README file.